### PR TITLE
fixed min-h to arbitrary values (rem)

### PR DESCRIPTION
**Pull request by grandehe4rt**
I have changed the css values from "common" px values to arbitrary values like "rem".

### CODE REVIEW

File: src/lib/theme/default.ts [hunk 1]
```diff
@@ -240,7 +240,7 @@ export default {
     },
   },
   carousel: {
-    base: 'relative h-full w-full min-h-56 sm:min-h-64 xl:min-h-80 2xl:min-h-96',
+    base: 'relative h-full w-full min-h-[3.5rem] sm:min-h-[4rem] xl:min-h-[5rem] 2xl:min-h-[6rem]',
```
Comment (tulup-conner) [inline src/lib/theme/default.ts:243]: Thank you! We should try to match with Flowbite itself and use the values provided by the original component here: https://flowbite.com/docs/components/carousel/

For example, the `2xl` breakpoint is set to `h-96` which translates to `24rem`

**Review by tulup-conner — CHANGES_REQUESTED** [review]

**Comment by grandehe4rt** [comment]
making clear that the carousels are now visible, i think my commit wasn't very clear.

File: src/lib/theme/default.ts [hunk 2]
```diff
@@ -240,7 +240,7 @@ export default {
     },
   },
   carousel: {
-    base: 'relative h-full w-full min-h-56 sm:min-h-64 xl:min-h-80 2xl:min-h-96',
+    base: 'relative h-full w-full h-56 sm:h-64 xl:h-80 2xl:h-96',
```
Comment (rluders) [inline src/lib/theme/default.ts:243]: I think that the we should replace the `h-full` and `w-full` with `h-min-full` and `w-min-full`, but we need to test. What do you think @tulup-conner?

**Review by rluders — CHANGES_REQUESTED** [review]

File: src/lib/theme/default.ts (hunk 2 above)
Comment (tulup-conner) [reply src/lib/theme/default.ts:243]: I think that could be fine as well.

**Comment by rluders** [comment]
Thanks @grandehe4rt

**Pull request merged**